activiti-cloud-acceptance-tests/activiti-cloud-acceptance-tests-core/src/main/java/org/activiti/cloud/acc/core/helper/Filters.java

```diff
@@ -36,6 +36,16 @@
         return filteredCollection;
     }
 
+    public static Collection<CloudProcessInstance> checkProcessInstances(Collection<CloudProcessInstance> resource, String processKey) {
```

Comment:
Needed a new method that accepts Collections instead of PagedModels.
The old method is still needed, since it's used by query and audit, services that still use manual clients